### DISCUSSION TITLE
Problem with authentication using personal authentication token

### DISCUSSION THREAD

**Discussion by Thanozz** [community/community#37261] [New to GitHub]
hi!
I am trying to push to an https site and i am using personal authentication token.
Then i read this:
remote: invalid username or password.
fatal: Authentication failed for 'https://github.....
I deleted the PAT and created a new one, but the problem is the same...

Any ideas?

**Comment by DSDmark** [comment]
Hey @Thanozz,

I'm not sure, but you could try these tips and also add screenshot if you can.

- Generate a [Personal Access Token](https://github.com/settings/tokens). Again and that time check that all boxes:- 


![Screenshot from 2022-10-26 12-44-18](https://user-images.githubusercontent.com/85744816/197961558-99f95c96-faa4-4d69-8a4a-297635526b17.png)

- Re-attempt the command you were trying, and use a Personal Access Token in the place of your password.
- Instead of **git pull** also try **git pull origin master**

If it is not work, then try to disable **2FA**

> 2FA is enabled and getting error remote: Invalid username or password. fatal: Authentication failed.

**Reply by airtower-luna** [reply]
Whether a token works or not doesn't depend on 2FA, don't recommend people disable it.

Pointing out the *repo* scope is correct, the *workflow* scope might also be needed if updating Actions workflows.

**Reply by DSDmark** [reply]
ok, got it.

**Reply by Thanozz** [reply]
Thank you for your replies...
I created a new PAT (with repo and workflow scope checked), but the problem is still there...
remote: Invalid username or password
fatal: Authentication failed for 'htpps://github.com.....'
I am really thinking of deleting everything and starting from scratch...

If someone has a clue why this is happening...?

**Reply by airtower-luna** [reply]
Is the repository actually *yours* (under your account)?

Also, exactly how are you entering the token? That part is missing in the output you're quoting.

**Reply by Thanozz** [reply]
The repository is under my account
I am entering the token using Shift+Ctrl+V

**Reply by DSDmark** [reply]
Hey @Thanozz,
can you add screenshot of your screen
